README.md

```diff
@@ -1 +1,185 @@
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Portfolio <a name="about-project"></a>
+
+>  
+
+**Portfolio** is project that shows some of the moile first designs that I have made at Microverse.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+     <li><a href="http://css.com/">CSS</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Header**
+- **headline section**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Code editor like Visual studio
+windows or Linux operating systems
+2GB RAM
+20G HDD
+
+### Setup
+
+Clone this repository to your desired folder using the following commands:
+
+
+  cd your-folder
+  git clone https://github.com/ausiemass/portfolio
+
+
+### Install
+
+Install this project with:
+
+
+  cd my-project
+  gem install
+
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+Shift + ctrl + R
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
+Shift + ctrl + T
+
+
+### Deployment
+
+You can deploy this project using:
+<ul>    
+<li>Windows</li>
+<li>Linux</li>
+</ul>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Austin Masamhiri**
+
+- GitHub: [@ausiemass](https://github.com/)
+- Twitter: [@ausiemass](https://twitter.com/home)
+- LinkedIn: [ausiemass](https://www.linkedin.com/feed/)
+
+👤
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **Desktop version**
+- [ ] **Contact Form**
+- [ ] **Mobile menu**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project please support by giving me 4 or 5 stars.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my collegues Nathaniel, Alice and Rachel who encouraged me tirelessly to produce this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
```

Comment:
- [ ] [Optional]👍 Great work on including a license section in your project! It's important to provide clear licensing information for your project. However, I noticed that the link you provided for the MIT license directs to a template instead of the actual license.

To resolve this issue, please make sure to update the link in the license section to point directly to the specific license file in your project repository. This will ensure that users can easily access the license terms and understand the permissions and restrictions associated with your project. You can use this [template ](https://github.com/microverseinc/readme-template) of the License provided by microverse.
![image](https://github.com/ausiemass/portfolio-austin/assets/55840999/92001449-3103-4f4c-8ea3-5730557f85f0)

Keep up the good work and ensure that your license section accurately reflects the licensing information for your project! 👍📜